config/environments/staging.rb

```diff
@@ -48,6 +48,9 @@
     reconnect_attempts: 1
   }
 
+  config.action_cable.url = "#{ENV['OFN_URL']}/cable"
+  config.action_cable.allowed_request_origins = [/http:\/\/*/, /https:\/\/*/]
```

Comment:
We will need this for production, too, right? And we should use the URL in the allowed origin, I think.

Reply:
Right will do. Then back to code review.